src/components/AnchorForCommentsOnly/BaseAnchorForCommentsOnly.js

```diff
@@ -68,7 +70,7 @@ function BaseAnchorForCommentsOnly(props) {
             <Tooltip text={props.href}>
                 <Text
                     ref={(el) => (linkRef = el)}
-                    style={StyleSheet.flatten([props.style, defaultTextStyle])}
+                    style={StyleSheet.flatten([props.style, defaultTextStyle, styles.cursorPointer])}
```

Comment:
@Nikhil-Vats I got a warning on the native app regarding the cursor style.

![Screenshot_1687350693](https://github.com/Expensify/App/assets/25520267/a71062be-8ca0-4b68-a872-43b8cb031254)

Reply:
@mollfpr fixed!